remote.php

```diff
@@ -94,7 +94,7 @@ public function saveData($page, $data, $summary) {
      * @throws RemoteAccessDeniedException
      * @throws RemoteException
      */
-    public function getSchema($schema) {
+    public function getSchema($schema = null) {
```

Comment:
I guess that makes the `if(!$schema) $schema = null;` call a few lines further down obsolete.

Reply:
not really, the code accepts also $schema=''.

but i simplified the code.

ps: it's really ugly practice to overwrite variables with different purpose. makes refactoring more difficult. the string method parameter `$schema` object in forach loop.

```
            $schemas = $this->hlp->getSchema($schema ?: null);
            foreach($schemas as $name => $schema) {
```

Reply:
Agreed  passing in `$schemaname` would probably make this more clear.